src/modules/appointment/models/appointment.entity.ts

```diff
@@ -177,4 +179,11 @@ export class AppointmentEntity {
   referralName?: string;
   referralId?: number;
 
+  @ManyToOne(type => EmployeeEntity, dentist => dentist.id)
+  @JoinColumn({ name: 'dentist_id' })
+  dentist: EmployeeEntity;
+
+  @ManyToOne(type => PatientEntity, patient => patient.id)
```

Comment:
do you need this?

Reply:
@imadiaz Yes, with that I can access and show the information to give it structure so that it only brings me what I need for each appointment, without that it would bring me the entire appointment object and separate the patient, dentist, etc. objects.

Reply:
got it, why did you comment the another properties?

Reply:
@imadiaz I'm going to check because that code was already commented in the file

Reply:
sure